.circleci/config.yml

```diff
@@ -28,36 +28,50 @@ jobs:
           command: | 
             git config --global user.name "Bodywork Machine Learning Ltd"
             git config --global user.email "info@bodyworkml.com"
+      - add_ssh_keys:
+            fingerprints:
+              - "3b:3f:26:dc:81:3b:6d:49:88:8e:da:f0:d5:9f:99:d6"
+              - "e2:88:27:a5:93:ed:d6:9f:1a:58:cf:07:a7:75:51:07"  
       - run:
           name: Installing Python dependencies
           command: pip install -r requirements_dev.txt
       - run: 
           name: Running tests
-          command: tox -e py38-unit_and_functional_tests,py38-static_code_analysis -- -s -vv
+          command: tox -e py38-unit_and_functional_tests,py38-static_code_analysis -- -s -vv --junitxml=~/tox/junit.xml
+      - store_test_results:
+          path: ~/tox/
       - run:
           name: Code Coverage upload
           command: bash <(curl -s https://codecov.io/bash)
-  run-k8s-integration-tests:
+  run-integration-tests:
     executor:
       name: aws-eks/python
       tag: "3.8"
     steps:
       - aws-eks/update-kubeconfig-with-authenticator:
           cluster-name: bodywork-dev
       - checkout
+      - add_ssh_keys:
+          fingerprints:
+            - "3b:3f:26:dc:81:3b:6d:49:88:8e:da:f0:d5:9f:99:d6"
```

Comment:
Which key does this correspond to?

The one with the fingerprint beginning `e2` is the GitLab one, but I don't recognise the one above.

Reply:
It's the Github key (look at SSH keys in CircleCI project settings). 

However I think it is better to change this to the Github Deploy key if it has access to the private repo as well.